entrypoint.sh

```diff
@@ -17,30 +17,81 @@ fi
 CMD="appium --log-no-colors --log-timestamp -pa /wd/hub --port $APPIUM_PORT --log $TASK_LOG --log-level $LOG_LEVEL $APPIUM_CLI $plugins_cli"
 #--use-plugins=relaxed-caps
 
+stop_video() {
+  local artifactId=$1
+  if [ -z ${artifactId} ]; then
+    echo "[warn] [Stop Video] artifactId param is empty!"
+    return 0
+  fi
+
+  if [ -f /tmp/${artifactId}.mp4 ]; then
+    ls -lah /tmp/${artifactId}.mp4
+    ffmpeg_pid=$(pgrep --full ffmpeg.*${artifactId}.mp4)
+    echo "[info] [Stop Video] ffmpeg_pid=$ffmpeg_pid"
+    kill -2 $ffmpeg_pid
+    echo "[info] [Stop Video] kill output: $?"
+
+    # wait until ffmpeg finished normally and file size is greater 48 byte! Time limit is 5 sec
```

Comment:
`and file size is greater 48 byte!` seems useless in current implementation as I don't see such validation anymore